lib/dynamodb/dynamodb_request_queue.c

```diff
@@ -666,7 +666,7 @@ dynamodb_request_queue_log(struct dynamodb_request_queue * Q,
  */
 void
 dynamodb_request_queue_setcapacity(struct dynamodb_request_queue * Q,
-    int capacity)
+    long capacity)
```

Comment:
I'm dubious.  The fact that the AWS API is documented as providing a `long` -- which AFAIK is not a defined type in JSON -- doesn't mean that their `long` matches the `long` on our machine.  In fact in general it can't, since we'll have 32-bit `long`s on some systems and 64-bit `long`s on others.

Reply:
Yes, there's no such thing as a `long` in JSON.  But there's two steps here:
- parse the JSON
- transmit that value to `dynamodb_request_queue_setcapacity()`.

The basic problem that sparked my interest in this is that we're truncating `long capr` to `int capacity`.  I thought that changing `capacity` to a `long` was a nicer solution than checking the value again `INT_MAX`, but I'm happy to do that instead if you prefer it that way?

Reply:
We can make it a `long` everywhere.  It just seemed like the explanation in the commit message of citing the AWS API didn't make sense.

Reply:
ok, I'll fix that in the next version.